src/styles/variables.scss

```diff
@@ -0,0 +1,23 @@
+@mixin cell-propertis ($cell-width, $cell-height, $bg-color) {
+  width: $cell-width;
+  height: $cell-height;
+  background-color: $bg-color;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Setting styling via `*` is irrelevant to the variable, so it's best to keep it in a more generic file.